translation/en-US.ts

```diff
@@ -177,6 +179,30 @@ export default {
   range_file: 'file',
   range_module: 'module',
 
+  //finance Page
+  bill_id: 'bill id',
+  bill_createAt: 'createAt',
+  bill_location: 'location',
+  bill_createBy: 'createBy',
+  bill_type: 'type',
+  bill_price: 'price',
+  bill_invoice: 'invoice',
+  bill_remark: 'remark',
```

Comment:
```suggestion
  bill_remark: 'note',
```
remark 一般指观点，这里就是一个备注